packages/core/useCloned/index.ts

```diff
@@ -23,6 +23,10 @@ export interface UseClonedReturn<T> {
    * Cloned ref
    */
   cloned: Ref<T>
+  /**
+   * IsCloneModified ref
+   */
+  isClonedModified: Ref<boolean>
```

Comment:
```suggestion
  isModified: Ref<boolean>
```

I think this would do already